README.md

```diff
@@ -20,62 +91,429 @@ Add the following to your `pubspec.yaml` file:
+                           )
+                         ),
+                         // no need for a scrolling gesture recognizer for embedded youtube videos so we only use VerticalDragGestureRecognizer when the iframe does not display embedded youtube videos
+                         gestureRecognizers: attributes['src'].contains("youtube.com/embed") ? null : [
+                           Factory(() => VerticalDragGestureRecognizer())
+                         ].toSet(),
+                         // no need to load other urls when displaying embedded youtube videos so we block url loading requests when this is the case
+                         shouldOverrideUrlLoading: (controller, request) async {
+                           if (attributes['src'].contains("youtube.com/embed")) {
+                             if (!request.url.contains("youtube.com/embed")) {
+                               return ShouldOverrideUrlLoadingAction.CANCEL;
+                             } else {
+                               return ShouldOverrideUrlLoadingAction.ALLOW;
+                             }
+                           } else {
+                             return ShouldOverrideUrlLoadingAction.ALLOW;
+                           }
+                         },
+                       ),
+                     );
+                   // if the src of the iframe is null then do not render anything
+                   } else {
+                     return Container(height: 0);
+                   }
+                 }
+               }
+           ),
+      ),
     );
+  }
+}
+```
+</details>
+
+More example usages and in-depth details available [here](https://github.com/Sub6Resources/flutter_html/wiki/All-About-customRender).
+
+### onImageError:
+
+A function that defines what the widget should do when an image fails to load. The function exposes the exception `Object` and `StackTrace` to use in your implementation.
+
+#### Example Usage - onImageError:
+
+```dart
+Widget html = Html(
+  data: """<img alt='Alt Text of an intentionally broken image' src='https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_92x30d'/>""",
+  onImageError: (Exception exception, StackTrace stackTrace) {
+    FirebaseCrashlytics.instance.recordError(exception, stackTrace);
+  },
+);
+```
+
+### onImageTap:
+
+A function that defines what the widget should do when an image is tapped.
+
+#### Example Usage - onImageTap:
+
+```dart
+Widget html = Html(
+  data: """<img alt='Google' src='https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_92x30dp.png' />""",
+  onImageTap: (String url) {
+    //open image in webview, or launch image in browser, or any other logic here
+  }
+);
+```
+
+### blacklistedElements:
+
+A list of elements the `Html` widget should not render. The list should contain the tags of the HTML elements you wish to blacklist.
+
+#### Example Usage - blacklistedElements:
+`h1` will render, but `h2` will not render
```

Comment:
Minor: can we think of a more realistic example? Maybe to never render iframes?

Reply:
Came up with a (hopefully) better example using iframe and video, let me know your thoughts.

Reply:
@tneotia Ah yeah that was like my first week maintain. This seems fine. Updating the wiki is a terrible pain, so this is just fine.

Reply:
It's _okay_ now, though the example still seems a bit contrived. Even just showing how to disabled any `<video>` or <audio>` tag support would have sufficed for me too. But it's fine.